src/hypertable.c

```diff
@@ -466,8 +466,8 @@ ts_hypertable_scan_with_memory_context(const char *schema, const char *table,
 									   bool tuplock, MemoryContext mctx)
 {
 	ScanKeyData scankey[2];
-	NameData schema_name = { { 0 } };
-	NameData table_name = { { 0 } };
+	NameData schema_name = { .data = { 0 } };
```

Comment:
clang-15 doesn't like double nested braces, and gcc 4.8.5 doesn't like single braces, so this is how we have to compromise to avoid warnings

Reply:
(for clang-15 this happens with -Wextra that we're probably going to enable soon)